conda_smithy/configure_feedstock.py

```diff
@@ -1499,6 +1499,20 @@ def _load_forge_config(forge_dir, exclusive_config_file):
             "win_64": "win_64",
             "osx_64": "osx_64",
         },
+        "os_version": {
+            "linux_64": "cos6",
+            "osx_64": None,
+            "win_64": None,
+            # Following platforms are disabled by default
+            "linux_aarch64": None,
+            "linux_ppc64le": None,
+            "linux_armv7l": None,
+            "linux_s390x": None,
+            # Following platforms are aliases of x86_64,
+            "linux": "cos6",
+            "osx": None,
+            "win": None,
```

Comment:
Let's not introduce these

Reply:
yeah I just realized they overwrote my input due to my sloppy handling. Will just remove them.

Reply:
Done in 46d7e1d. Everything looks good to me.